Source/UZKArchive.h

```diff
@@ -114,6 +114,10 @@ typedef NS_ENUM(NSInteger, UZKErrorCode) {
      *  The CRC given up front doesn't match the calculated CRC
      */
     UZKErrorCodePreCRCMismatch = 114,
+    /**
+     *  The CRC given up front doesn't match the calculated CRC
+     */
+    UZKErrorCodeFileHandleCreate = 115,
```

Comment:
Please update the documentation to reflect when this new error code is used. Also, please add it to `+[UZKArchive errorNameForErrorCode:]`, which produces the error text used in the `NSError`